### DISCUSSION TITLE
Any tips for domain name migration/change (FAST)?

### DISCUSSION THREAD

**Discussion by chpetit-capgemini** [GoogleCloudPlatform/cloud-foundation-fabric#2068] [Q&A] [answered]
First of all thank you for all your work and time.

My company is considering changing the domain name and releasing the old one we were using to set up a training landing zone. We use Fabric FAST to configure our landing zone. Do you have any tips to share before forcing domain name change in Terraform files. Some steps to pay attention to? What should I do first in Fabric FAST? Traps to avoid?

**Comment by ludoo** [comment]
Are you changing the org domain, and have you deployed FAST already?

**Reply by chpetit-capgemini** [reply]
Hello, thank you for your answer.
Not yet, for the moment I am in the impact study phase.

**Reply by ludoo** [reply, marked as answer]
Ok, so it should not change anything from the FAST PoV. Right now FAST depends on groups defined in the Cloud Identity associated with the org, but that is changing soon with PR #2064 

Once that is in, you can use any type of principal where FAST expects a group, so you could have `group:gcp-organization-admins@other.domain` or even use `principal:` and `principalSet` to bring over external identities via Workforce Identity Federation.

**Reply by juliocc** [reply]
Also, keep in mind that changing the domain is a Cloud Identity/Workspace [process](https://support.google.com/cloudidentity/answer/7009324?hl=en). It cannot be done from Terraform.

**Reply by chpetit-capgemini** [reply]
Thank you all,

@juliocc: yes!
